src/imex/transfer.rs

```diff
@@ -380,6 +290,66 @@ pub async fn get_backup(context: &Context, qr: Qr) -> Result<()> {
     res
 }
 
+pub async fn get_backup2(
+    context: &Context,
+    node_addr: iroh_net::NodeAddr,
+    auth_token: String,
+) -> Result<()> {
+    let relay_mode = RelayMode::Disabled;
+
+    let endpoint = Endpoint::builder().relay_mode(relay_mode).bind(0).await?;
+
+    let conn = endpoint.connect(node_addr, b"/deltachat/backup").await?;
+    let (mut send_stream, mut recv_stream) = conn.open_bi().await?;
+    info!(context, "Sending backup authentication token.");
+    send_stream.write_all(auth_token.as_bytes()).await?;
+    send_stream.flush().await?;
```

Comment:
Do you need this flush?  It won't do any harm probably, but I'm a bit surprised seeing it.

Reply:
Did you observe this?  I'm fairly sure this flush is a no-op in Quinn: https://github.com/quinn-rs/quinn/blob/main/quinn/src/send_stream.rs#L274-L276